package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "prebuild": "(rm -r build && mkdir build) || mkdir build",
-    "build": "webpack && cp -r i18n build/i18n && cp -r core build/core && cp -r vendor build/vendor && cp -r views build/views && cp -r styles build/styles && find ./components -iname \"*.html\" -exec rsync -R '{}' ./build/ \\;",
+    "build": "webpack && cp package.json build && cp -r i18n build/i18n && cp -r core build/core && cp -r vendor build/vendor && cp -r views build/views && cp -r styles build/styles && find ./components -iname \"*.html\" -exec rsync -R '{}' ./build/ \\;",
```

Comment:
What does this help you achieve?

Reply:
Me neither, I added it because the script for bundling apps seems to look for it as part of the bundling process. I can remove it again as it seems this will work without it as well.

Reply:
You can perhaps ave a look at this line in git.js(114) in dhis2-core: 
const pkg_name = require(path.join(clone_path, 'package.json')).name

Reply:
Okay. I would remove this `cp` command if everything works without it as well, but you can be the judge of it :)

Reply:
I see most other apps are including package.json, so I'll leave this in.